src/server/webpack-config.js

```diff
@@ -33,26 +33,34 @@ const productionEnvPlugin = new webpack.DefinePlugin({
 });
 
 
+const relayPluginQuery = `plugins[]=${ fsPath.join(__dirname, '../relay/babel-relay-plugin') }`;
+
 const babelLoader = (extension, isRelayEnabled) => {
   const loader = {
     // See: https://github.com/babel/babel-loader#options
-    loader: 'babel',
+    loaders: ['babel'],
     test: extension,
     exclude: /(node_modules|bower_components)/,
-    query: {
-      plugins: [],
-    },
   };
 
   // Add optional plugins.
   if (isRelayEnabled) {
-    loader.query.plugins.push(fsPath.join(__dirname, '../relay/babel-relay-plugin'));
+    loader.loaders[0] += relayPluginQuery;
```

Comment:
Hey @frederickfogerty - have you verified that the GraphQL/Relay stuff will works with these changes?